src/common/pure/ZeroApprovalWarning/ZeroApprovalWarning.tsx

```diff
@@ -0,0 +1,31 @@
+import { HashLink } from 'react-router-hash-link'
+import { WarningCard } from '../WarningCard'
+import styled from 'styled-components/macro'
+import { Currency } from '@uniswap/sdk-core'
+
+const Link = styled(HashLink)`
+  text-decoration: underline;
+`
+
+interface ZeroApprovalWarningProps {
+  currency: Currency | undefined
+}
+
+// TODO: Replace with FAQ link once available.
+const faqLink = undefined
+
+export function ZeroApprovalWarning({ currency }: ZeroApprovalWarningProps) {
+  const symbol = currency?.symbol?.toUpperCase()
+
+  if (!symbol) {
+    return <></>
+  }
+
+  return (
+    <WarningCard>
+      <strong>Note:</strong> {symbol} specifically requires 2 approval transactions. The first resets your spending cap
+      to 0, and the second sets your desired spending cap. To avoid this in the future, set your spending cap to CoW
+      Swap's recommended default amount. {faqLink && <Link to={faqLink}>Learn more</Link>}
+    </WarningCard>
+  )
+}
```

Comment:
This looks an awful lot with the new component added for tx bundling https://github.com/cowprotocol/cowswap/blob/a17b28d6f1af2c65e6464aa250e78996074bb6c2/src/common/pure/InlineBanner/index.tsx#L65

Could it be re-used/merged?

Reply:
Would need to check. Keeping ZeroApprovalWarning still makes sense, I think, but it could use InlineBanner instead of WarningBanner.